src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -544,6 +526,26 @@ export default class RadarTargetModel {
         return [true, 'ADJUST HALO'];
     }
 
+    /**
+     * Initialize the value of the scratchpad
+     *
+     * @for RadarTargetModel
+     * @method setDefaultScratchpad
+     * @private
+     * @chainable
+     */
+    setDefaultScratchpad(aircraftModel) {
+        if (aircraftModel.isDeparture()) {
+            this.scratchPadText = aircraftModel.fms.getFlightPlanEntry();
+
+            return [true, 'RESET SCRATCHPAD'];
+        }
+
+        this.scratchPadText = aircraftModel.destination.substr(1, 3);
```

Comment:
We really don't need to _pass_ the aircraftModel like this, we should instead just access it with `this.aircraftModel`, as used in other methods of this class.

This change is the reason some tests are now failing-- because the tests aren't passing the aircraftModel, they're expecting you to retrieve it from the RadarTargetModel.aircraftModel.